core/src/main/java/org/kohsuke/stapler/framework/io/LargeText.java

```diff
@@ -340,24 +339,6 @@ private void setContentTypeImpl(StaplerResponse2 rsp) {
         rsp.setContentType("text/plain;charset=UTF-8");
     }
 
-    protected Writer createWriter(StaplerRequest2 req, StaplerResponse2 rsp, long size) throws IOException {
-        if (ReflectionUtils.isOverridden(
-                LargeText.class, getClass(), "createWriter", StaplerRequest.class, StaplerResponse.class, long.class)) {
-            return createWriter(
-                    StaplerRequest.fromStaplerRequest2(req), StaplerResponse.fromStaplerResponse2(rsp), size);
-        } else {
-            return rsp.getWriter();
-        }
-    }
-
-    /**
-     * @deprecated use {@link #createWriter(StaplerRequest2, StaplerResponse2, long)}
-     */
-    @Deprecated
-    protected Writer createWriter(StaplerRequest req, StaplerResponse rsp, long size) throws IOException {
-        return rsp.getWriter();
-    }
```

Comment:
As noted in the core PR, this is not overridden (by `AnnotatedLargeText` or anything else), and the `size` parameter is ignored.